apps/nouns-camp/src/utils/voters.js

```diff
@@ -23,8 +23,5 @@ export const buildFeed = (delegate, { proposals, candidates }) => {
 
   const items = [...propFeedItems, ...candidateFeedItems];
 
-  return arrayUtils.sortBy(
-    { value: (i) => i.blockNumber, order: "desc" },
-    items
-  );
+  return arrayUtils.sortBy({ value: (i) => i.timestamp, order: "desc" }, items);
```

Comment:
Just curious, are we missing block number for something?

Reply:
hm I remember specifically changing this for a reason, but I can't remember why now 🤔 

looking through the code, it does seem that the timestamp and block numbers co-exist on all events. maybe they didn't before?